.github/workflows/build-yocto.yml

```diff
@@ -60,7 +60,7 @@ jobs:
       run: |
         cd Build
         ./run.sh -c /home/github/rzg2l-cache
-        cp output/images/smarc-rzv2l/*.wic.bz2 /home/github
+        cp output/images/smarc-rzg2l/*.wic.bz2 /home/github
```

Comment:
Can this `rzg2l` or `rzv2l` be a variable? When does it need to change? How does it relate to the build?

Reply:
This is a G2L repo. It doesn't change.
It moves/copies the wic.bz2 image to a more accessible location for uploading to release artifacts.

Reply:
sounds good!